traffic_portal/app/src/common/modules/table/capabilityUsers/TableCapabilityUsersController.js

```diff
@@ -17,11 +17,14 @@
  * under the License.
  */
 
-var TableCapabilityUsersController = function(capability, capUsers, $scope, $state, locationUtils) {
+var TableCapabilityUsersController = function(capability, capUsers, $controller, $scope, $state, locationUtils) {
 
-	$scope.capability = capability;
+	// extends the TableUsersController to inherit common methods
+	angular.extend(this, $controller('TableUsersController', { users: capUsers, $scope: $scope }));
 
-	$scope.capUsers = capUsers;
+	let capUsersTable;
+
+	$scope.capability = capability[0];
```

Comment:
more of just for my understanding but why did this go to indexing of the capability when setting it in the scope?

Reply:
the call to get a capability looked like this:

`GET /api/capabilities?name=foo` which returns an array of 1 so this was a bug that i fixed with this PR as you need to fetch it with the index.

Reply:
ahh make sense 👍